### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by KyleJu**
Update installation instructions
 - Specific the directories 
 - Use an older node version
 - install pip for python 2
 - Use python2 to install `requirements.dev.txt`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,23 @@ Chrome Platform Status
 
 ### Installation
 1. Before you begin, make sure that you have a java JRE (version 8 or greater) installed. JRE is required to use the DataStore Emulator.
-1. Install global CLIs
+1. Install global CLIs in the home directory
     1. [Google App Engine SDK for Python](https://cloud.google.com/appengine/docs/standard/python3/setting-up-environment). Make sure to select Python 3.
     1. pip, node, npm.
```
Comment (dlaliberte) [inline README.md:16]: Installing pip is problematic... See below.  
Add nvm, since it is used for using node 10.

File: README.md [hunk 2]
```diff
@@ -11,16 +11,23 @@ Chrome Platform Status
...
     1. Gulp `npm install --global gulp-cli`
-1. Install npm dependencies `npm ci`
-1. Create a virtual environment.
+1. We recommend using an older node version, e.g. node 10
+    1. Use `node -v` to check the default node version
+    2. `nvm use 10` to switch to node 10
+3. cd into the Chromestatus repo and install npm dependencies `npm ci`
+4. Create a virtual environment.
     1. `apt install python3.9-venv`
```
Comment (dlaliberte) [inline README.md:23]: 'sudo' is required for apt install python3.9-venv

File: README.md [hunk 3]
```diff
@@ -11,16 +11,23 @@ Chrome Platform Status
...
-1. Install global CLIs
+1. Install global CLIs in the home directory
     1. [Google App Engine SDK for Python](https://cloud.google.com/appengine/docs/standard/python3/setting-up-environment). Make sure to select Python 3.
     1. pip, node, npm.
     1. Gulp `npm install --global gulp-cli`
-1. Install npm dependencies `npm ci`
-1. Create a virtual environment.
+1. We recommend using an older node version, e.g. node 10
+    1. Use `node -v` to check the default node version
+    2. `nvm use 10` to switch to node 10
```
Comment (jrobbins) [inline README.md:20]: I have node 12 installed and it seems to work.  If we could bump this up to 12, that might be better somehow.

File: package.json [hunk 4]
```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "deps": "python3 -m pip install -t lib -r requirements.txt --upgrade; cd py2; python3 -m pip install -t lib -r requirements.txt --upgrade",
-    "dev-deps": "python3 -m pip install --no-deps -r requirements.dev.txt",
+    "dev-deps": "python2 -m pip install --no-deps -r requirements.dev.txt",
```
Comment (jrobbins) [inline package.json:11]: Do we need this for both py2 and py3?  I am not sure.

File: README.md (hunk 3 above)
Comment (KyleJu) [inline README.md:20]:
```suggestion
    2. `nvm use 12` to switch to node 12
```

File: README.md [hunk 5]
```diff
@@ -11,16 +11,23 @@ Chrome Platform Status
...
+3. `cd` into the Chromestatus repo and install npm dependencies `npm ci`
+4. Create a virtual environment.
     1. `apt install python3.9-venv`
```
Comment (KyleJu) [inline README.md:23]:
```suggestion
    1. `sudo apt install python3.9-venv`
```

File: package.json (hunk 4 above)
Comment (KyleJu) [reply package.json:11]: My guess is it should be py2 only. I have removed `npm run dev-deps` from python3 CI and lets see if the tests pass

Comment (jrobbins) [reply package.json:11]: Looks like you were right.

**Review by jrobbins — APPROVED** [review]

**Pull request merged**